src/FastNorth/PropertyMapper/Processor/Links.php

```diff
@@ -23,13 +23,14 @@ class Links extends Processor
     public function process($from, &$to, MapInterface $map)
     {
         foreach ($map->getLinks() as $link) {
-            if ($link->hasTransformer()) {
-                $value = $link->getTransformer()->transform(
-                    $this->propertyAccess->getValue($from, $link->getFrom()),
-                    $from
-                );
-            } else {
+            if ($this->propertyAccess->isReadable($from, $link->getFrom())) {
```

Comment:
Potential BC break here as it will always map something that doesn't exist instead of throwing the previous exceptions